isis/tests/FunctionalTestsIsisImportMerMI.cpp

```diff
@@ -0,0 +1,271 @@
+#include <iostream>
+#include <time.h>
+
+#include <QRegExp>
+#include <QString>
+#include <nlohmann/json.hpp>
+
+#include "Fixtures.h"
+#include "md5wrapper.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+
+#include "isisimport.h"
+#include "TestUtilities.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using json = nlohmann::json;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/isisimport.xml").expanded();
+
+
+TEST_F(TempTestingFiles, FunctionalTestIsisImportMerMICaseOne){
```

Comment:
Is there a better name for these. If it is just that way from history that ok, but if during the conversion we find out how "CaseOne" is different from "CaseTwo" we should name them something with meaning.

Reply:
I don't know of a better name for these tests, maybe @acpaquette has a better idea of naming these tests.